ui/src/apps/keyvis/ToolBar.tsx

```diff
@@ -18,133 +17,147 @@ export interface IKeyVisToolBarProps {
-                  onClick={() => {
-                    this.handleBrightLevelChange('up')
-                  }}
-                />
-              </Button.Group>
-              Set Brightness */}
-            {/* </Menu.Item> */}
-
-            <Menu.Item name="resetZoom" onClick={this.props.onResetZoom}>
-              <Icon name="zoom-out" />
-              Reset Zoom
-            </Menu.Item>
-
-            <Menu.Item name="toogleBrush" color="green" onClick={this.props.onToggleBrush} active={isOnBrush}>
-              <Icon name="zoom-in" />
-              Zoom In
-            </Menu.Item>
-
-            <Menu.Item name="autoUpdate" color="green" active={isAutoFetch} onClick={this.handleAutoFetch}>
-              <Icon name="refresh" />
-              Auto Update
-            </Menu.Item>
-
-            <Menu.Item>
-              <Icon name="clock outline" />
-
-              <Dropdown
-                placeholder="Quick range"
-                onChange={this.handleDateRange}
-                options={DateRangeOptions}
-                value={dateRange}
+      <div className="PD-KeyVis-Toolbar">
+        <Dropdown
+          overlay={
+            <div id="brightness-overlay">
+              <div className="PD-Cluster-Legend" />
+              <Slider
+                style={{ width: 360 }}
+                defaultValue={0}
+                min={-6}
+                max={6}
+                step={0.1}
+                onChange={value => this.handleBrightLevel(value as number)}
               />
-            </Menu.Item>
-
-            <Menu.Item>
-              <Icon name="chart area" />
-              <Dropdown
-                placeholder="Metric"
-                onChange={this.handleMetricChange}
-                options={MetricOptions}
-                value={metricType}
-              />
-            </Menu.Item>
-          </Menu.Menu>
-        </Menu>
-      </>
+            </div>
+          }
+          trigger={['click']}
+          onVisibleChange={this.handleBrightnessDropdown}
+          visible={this.state.brightnessDropdownVisible}
+        >
+          <Button icon="bulb">
+            调整亮度
+            <Icon type="down" />
+          </Button>
+        </Dropdown>
+
+        <div className="space" />
+
+        <Button.Group>
+          <Button
+            onClick={this.props.onToggleBrush}
+            icon="arrows-alt"
+            type={isOnBrush ? 'primary' : 'default'}
+          >
+            框选
+          </Button>
+          <Button onClick={this.props.onResetZoom}>重置</Button>
+        </Button.Group>
+
+        <div className="space" />
+
+        <Button
+          onClick={this.handleAutoFetch}
+          icon="sync"
+          type={isAutoFetch ? 'primary' : 'default'}
+        >
+          自动刷新
+        </Button>
+
+        <div className="space" />
+
+        <Input.Group compact style={{ width: 142 }}>
+          <div className="select-icon">
```

Comment:
I prefer to avoid duplicating ant-design's style as much as possible. As a solution for this, how about: https://github.com/ant-design/ant-design/issues/5744#issuecomment-550399711

Reply:
Awesome, works great!